pillar/edx/ansible_vars.sls

```diff
@@ -370,6 +371,7 @@ edx:
     EDXAPP_GRADE_STORAGE_TYPE: S3
     EDXAPP_JWT_SECRET_KEY: {{ salt.vault.read('secret-{business_unit}/{env}/edxapp-jwt-secret-key'.format(env=environment, business_unit=business_unit)).data.value }}
     EDXAPP_LMS_BASE: "{{ LMS_DOMAIN }}"
+    EDXAPP_LMS_MAX_REQ: 1000
```

Comment:
What does this do exactly? Restart gunicorn after 1000 requests?

Reply:
Yes, it causes the workers to restart after 1000 requests so that they can free up the memory that they are consuming, thereby working around the memory leak. Because it's the workers restarting and not the entire process it avoids the issue of a user hitting a 502 error from not having any backend listeners available.